components/wordPrompt.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.nextWords.length != 0) {
+        return {
+          ...lastWords,
+          currentWord: lastWords.nextWords.shift(),
+        };
+      } else {
+        return {
```

Comment:
maybe we can just do a `fetchWords` here, instead of letting the list go empty

Reply:
I've set it now so that when the word count goes below 3, it'll call fetchWords